nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -230,27 +230,55 @@ xkb_symbols &quot;us-greek&quot;
   key &lt;LatZ&gt; { [ z, Z, Greek_zeta  ] };
 };
 </programlisting>
+  <note>
   <para>
-   To install the layout, the filepath, a description and the list of
-   languages must be given:
+   The longest layout name will be the width of the layout indicator
```

Comment:
i can't answer. it's not the file name (because the layout from the file "us" is called "en"), but that file doesn't contain the words "en" displayed in the widget.

i'll just delete this note.